src/elm/clinical.ts

```diff
@@ -121,11 +130,12 @@ export class CodeDef extends Expression {
   systemName: string;
   display?: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.CodeDef) {
+    // @ts-ignore
```

Comment:
Same as initial comment with more explanation of ts-ignore and disabling the lint rule